tests/integration/nft/simulation/operations_test.go

```diff
@@ -42,7 +42,7 @@ type SimTestSuite struct {
 
 func (suite *SimTestSuite) SetupTest() {
```

Comment:
if we are moving simulation tests from module to tests we should create their own folder. Maybe we leave this until we know what to do with these tests. 

Moving to tests is kind of moving the problem instead of fixing the problem with tests. Same thing goes for integration tests. It's something @julienrbrt and @likhita-809 are tackling and it would be nice to see these tests moved back to their modules in the future.